README.md

```diff
@@ -1,3 +1,4 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15535780&assignment_repo_type=AssignmentRepo)
 # SE_Day1
```

Comment:
Example of a Vague Prompt

Vague: "Tell me about science." Specific: "Explain the process of photosynthesis in plants."

Without Context: "What are the benefits?" With Context: "What are the benefits of daily exercise for mental health?

The improved prompt is more effective because:

vague terms were avoided and being precise in our wording helps the language model understand exactly what we are asking.

it sets the scene and helps the model understand the background, leading to more accurate and relevant responses.